data/campaigns/Liberty/scenarios/05_Hide_and_Seek.cfg

```diff
@@ -46,10 +46,10 @@
         side=2
         canrecruit=yes
         controller=ai
-        ai_algorithm=idle_ai
         user_team_name=_"Asheviere"
         team_name=bad_guys
         [ai]
+            ai_algorithm=idle_ai
```

Comment:
Dunno. Just swatted the bug. Didn't question the original intention. [ai] is PFM to me ATM